### PR TITLE
Feature/knotx#121 knot skeleton

### PR DESCRIPTION
**Pull request by Shaihuludus**

### CODE REVIEW

File: knotx-knots/knotx-knot-view/src/main/java/com/cognifide/knotx/knot/view/ViewKnotVerticle.java [hunk 1]
```diff
@@ -61,17 +59,36 @@ public void start() throws Exception {
...
+  }
+
+  @Override
+  protected ViewKnotConfiguration initConfiguration(JsonObject config) {
+    return new ViewKnotConfiguration(config);
+  }
+
+  @Override
+  protected void handle(Message result, Handler handler) {
+    //Not used in this knot
```
Comment (Skejven) [inline knotx-knots/knotx-knot-view/src/main/java/com/cognifide/knotx/knot/view/ViewKnotVerticle.java]: Isn't here line `66` `handle()` usage?

**Comment by Skejven** [comment]
Please do not merge this task yet.

File: knotx-knots/knotx-knot-view/src/main/java/com/cognifide/knotx/knot/view/ViewKnotVerticle.java (hunk 1 above)
Comment (tomaszmichalak) [inline knotx-knots/knotx-knot-view/src/main/java/com/cognifide/knotx/knot/view/ViewKnotVerticle.java]: fixed.

**Comment by Shaihuludus** [comment]
I hereby agree to the terms of the Knot.x Contributor License Agreement.

**Pull request merged**